PDK_Abstraction/FinFET14nm_Mock_PDK/FinFET_Mock_PDK_Abstraction_v1.json

```diff
@@ -0,0 +1,436 @@
+{
+"_comment":"The rules defined in this file are discussed in FinFET_MOCK_PDK.ppt",
+
+"Abstraction": [
+
+{
+"Layer": "Poly",
+"LayerNo": 1,
+"Direction": "V",
+"Width": [14],
+"Space": [66],
+"MinL": [60],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "Fin",
+"LayerNo": 3,
+"Direction": "H",
+"Width": [10],
+"Space": [32],
+"MinL": [100],
+"MaxL": ["NA"],
+"End-to-End": ["NA"],
+"Offset": [0]
+},
+
+{
+"Layer": "M1",
+"LayerNo": 6,
+"Direction": "V",
+"Color": 2,
```

Comment:
I see you dropped the "Type" field. Are you relying on the Layer Names to determine type instead?

Reply:
Yes, "Type" field is confusing; now, a layer can be used for signal /power routing purpose.

Reply:
So layers are generic & can route both power & signal... Its upto the person designing the primitive to pick the right width if multiple widths are available. Feels like the right place for this anyway. Closing this thread.

Reply:
Yes, you are right.